versioned_docs/version-2.6/pages-for-subheaders/authentication-config.md

```diff
@@ -109,3 +109,23 @@ If you need to reconfigure or disable then re-enable a provider that had been pr
 is logged in to Rancher as an external user, not the local admin.
 
 :::
+
+## Disabling an auth provider
+
+When you disable an auth provider, Rancher deletes all resources associated with it, including secrets,
+global role bindings, cluster and role template bindings, and users who had never been local users in Rancher.
```

Comment:
```suggestion
When you disable an auth provider, Rancher deletes all related resources, such as:

- Secrets.
- Global role bindings.
- Cluster bindings 
- Role template bindings.
- Users associated with the provider, who never logged in as local users to Rancher.
```

I arranged this as a list for legibility. Is the last item rephrased accurately?

Reply:
Will use a list! The last item refers to users who never were local users, but only external users.

Reply:
Updated the suggestion to read "External users..."